persistence-modules/hibernate-mapping/src/main/java/com/baeldung/hibernate/wherejointable/Group.java

```diff
@@ -0,0 +1,85 @@
+package com.baeldung.hibernate.wherejointable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+
+@Entity(name = "e_group")
+public class Group {
+
+    @Id
+    @GeneratedValue
+    private Long id;
+
+    private String name;
+
+    @ManyToMany(mappedBy = "groups")
+    private List<User> users = new ArrayList<>();
+
+    public Group() {
+
+    }
+
+    public Group(String name) {
+        this.name = name;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public List<User> getUsers() {
+        return users;
+    }
+
+    public void setUsers(List<User> users) {
+        this.users = users;
+    }
+
+    @Override
+    public String toString() {
+        return "Group [name=" + name + "]";
+    }
+
+    @Override
+    public int hashCode() {
+        final int prime = 31;
+        int result = 1;
+        result = prime * result + ((name == null) ? 0 : name.hashCode());
+        return result;
+    }
+
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
Although this is generated code, it probably should use `{}` with if statements. Again, this looks like a pre-Java 7 template for creating an equals method.

Reply:
Same for here, refactoring them to Java 7+ way.